### ISSUE TITLE
Log errors on query failure

### ISSUE THREAD

**Issue by omid** [fmeringdal/nettu-scheduler#17]
For some of the queries, if the query fails (eg. db connection failure) it's not possible to track down the error. The code below is one example: https://github.com/fmeringdal/nettu-scheduler/blob/master/scheduler/crates/infra/src/repos/account/postgres.rs#L125

**Comment by fmeringdal** [comment]
You are correct. Some queries have error logs enabled, like [this one](https://github.com/fmeringdal/nettu-scheduler/blob/master/scheduler/crates/infra/src/repos/event/event/postgres.rs#L166-L169). I think that is a copy paste job needed to have that error logging enabled for all queries.
I also think if `RUST_LOG=error` env var is set then sqlx error logging will be enabled, but those logs will be too noisy and hard to track down the error origin from.
